### DISCUSSION TITLE
itp parser

### DISCUSSION THREAD

**Discussion by aspitaleri** [MDAnalysis/mdanalysis#4600] [General]
Hi folk,
I have been reading user guide and searching for it, but I couldn't find it. In few words, is it possible to modify a itp file in case we remove an atom (or more) from the itp file (and from the gro file respectively)? Removing an atom (or more) everything then should be renumbered accordingly. Hope I was clear.
Best

**Comment by orbeckst** [comment]
MDAnalysis has no real support for topology manipulation and that's on purpose because it's complicated and better left to specialized tools. Therefore, there's no support for changing an ITP file. Perhaps [parmed](https://parmed.github.io/ParmEd/html/index.html) (MDAnalysis has a [parmed converter](https://docs.mdanalysis.org/stable/documentation_pages/converters/ParmEd.html)) or [gromologist](https://gitlab.com/KomBioMol/gromologist) can do what you want? 

You _can_ use MDAnalysis to build different systems using [Merge()](https://docs.mdanalysis.org/stable/documentation_pages/core/universe.html#MDAnalysis.core.universe.Merge) (see also [User Guide: constructing universes](https://userguide.mdanalysis.org/stable/examples/constructing_universe.html). In short, you select the parts you want as atomgroups and merge all the groups you want, leaving out what you want to delete.
